src/transaction/transaction.ts

```diff
@@ -194,6 +194,9 @@ export const sendTransaction = async function (
   const { transaction, from, sendParams } = send
   const { skipSending, skipWaiting, fee, maxFee, suppressLog, maxRoundsToWaitForConfirmation, atc } = sendParams ?? {}
 
+  if (sendParams?.firstValid) transaction.firstRound = sendParams.firstValid
```

Comment:
Should we also add it to sendAtomicTransactionComposer, from memory it's a different code path, but a bit hazy on it.

Also, we should add this to the legacyTransactionBridge in #287

Reply:
Well we can't really add it to `sendAtomicTransactionComposer` because it's a transaction-specific field. Setting it on `sendAtomicTransactionComposer` would basically force us to update ALL the transactions with the specified lv/fv which we might not want.

Reply:
Yeah, that makes sense, I think I'm just getting confused about the code paths